### ISSUE TITLE
Exits with: Wrong message type! Expected ActionResponseMessage

### ISSUE THREAD

**Issue by TylerRick** [burke/zeus#507]
## Description of Problem

`zeus start` exits with error:

```
slavenode.go:215: EOF
slavenode.go:223: [development_environment] Wrong message type! Expected ActionResponseMessage, got: 
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4cd944]
```

zeus had been working just beautifully (thank you) for quite a few months on `zeus 0.13.4.pre2` and `ruby 2.1.0`, but as soon as I upgraded to `ruby 2.2.0` and `zeus 0.15.3`, it started giving this error.

## System details

* **`uname -a`**: `Linux hostname 3.13.0-43-generic #72-Ubuntu SMP Mon Dec 8 19:35:06 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux`

* **`ruby -v`**: `ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-linux]`

## Steps to Reproduce

1) `zeus start` in my existing rails project

## Observed Behavior

* Exits with [this error and backtrace](https://gist.github.com/TylerRick/d5e2a57c95b354ddec5d)

## Expected Behavior

* zeus starts successfully

**Comment by TylerRick** [comment]
When I try it again, I sometimes get this error message instead:

```
The FS watcher process crashed: signal: aborted (core dumped)
The FS watcher process crashed: signal: aborted (core dumped)
```

I then notice that some zeus processes are still running so I kill them with
`psgrep  zeus | awk '{ print $2 }' | xargs kill`
and try again and get the original error again.

**Comment by TylerRick** [comment]
I've tried uninstalling the zeus gem and then re-installing each older version back to 0.13.4.pre2 and it still gives the same error (`runtime error: invalid memory address or nil pointer dereference`) so it appears that the zeus version doesn't make a difference.  The only variable that should have changed, then, is the Ruby version, from 2.1 to 2.2.

**Comment by TylerRick** [comment]
Update: Since upgrading to Ruby 2.2, I can't even start the app with a regular `rails c`.  I guess I shouldn't expect `zeus start` to work if I can't even start the app without zeus.  I will update this issue if the zeus issue still exists after resolving these other dependency issues...

(Current error is:
`/home/tyler/.rvm/rubies/ruby-2.2.0/bin/ruby: symbol lookup error: .../extensions/x86_64-linux/2.2.0/pg-0.14.1/pg_ext.so: undefined symbol: rb_thread_select`)

**Comment by latortuga** [comment]
That error you posted looks to me like you need to reinstall your pg gem for the new ruby version.

**Comment by TylerRick** [comment]
Yep, re-installing pg gem solved that error, and I finally got the app to start using `rails s` and `rails c`.  So back to zeus...  

Still giving this error when I try to start it:
```
The FS watcher process crashed: signal: aborted (core dumped)
The FS watcher process crashed: signal: aborted (core dumped)
```

I guess I should try replicating the issue in a brand-new rails project to completely rule out my app/dependencies being the problem...
